examples/app.R

```diff
@@ -215,6 +244,20 @@ card_demo <- function() {
     )
   )
 }
+
+msgbox_demo <- function() {
+  div(
+    h1(class="ui header", id="messagebox", "Messagebox"),
+    demo(messagebox(header = "Main header", content = "text")),
```

Comment:
Just the question for future: Is `messagebox` proper name according to our naming guide? Shouldn't that be `message_box`?

Reply:
I will add issue for this